usage/rest-server/src/test/java/brooklyn/rest/HaHotCheckTest.java

```diff
@@ -43,75 +43,61 @@
 
 public class HaHotCheckTest extends BrooklynRestResourceTest {
 
-    private ManagementContextMock mgmtMock;
-
-    @Override
-    @BeforeClass(alwaysRun = true)
-    public void setUp() throws Exception {
-        mgmtMock = new ManagementContextMock();
-        super.setUp();
-    }
-
-    @Override
-    @AfterClass(alwaysRun = true)
-    public void tearDown() throws Exception {
-        super.tearDown();
-    }
-
     @BeforeMethod(alwaysRun = true)
-    public void setUpMethod() {
-        mgmtMock.setState(ManagementNodeState.MASTER);
-    }
+    public void setUp() throws Exception { super.setUp(); }
 
+    @AfterMethod(alwaysRun = false)
+    public void tearDown() throws Exception { super.tearDown(); }
```

Comment:
Are these needed, won't the superclass methods be picked up if these are removed? Missing `@Override`.